rust/otel-arrow-rust/src/pdata/otlp/mod.rs

```diff
@@ -0,0 +1,30 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! This module contains traits and utilities for OTLP (OpenTelemetry Protocol) message types.
+
+// Re-export derive macros (required for generated code)
+pub use otlp_derive::Message;
+pub use otlp_derive::qualified;
+
+// Include tests
+#[cfg(test)]
+mod tests;
+
+/// Message is a trait for OTLP protocol buffer message types.
+pub trait Message {
+    /// The associated builder type for this message
+    type Builder;
+
+    /// Creates a new builder for this message type
+    fn builder() -> Self::Builder;
```

Comment:
I haven't completely understood the implementation details of the procedural derive macro, but I was expecting this method to be called somewhere. In the tests, I see only the constructor methods being called.

Is `Message::builder()` expected to be called once you have the arrow related work done?

Reply:
This was dead code. I removed it. Good catch, thanks.